docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -0,0 +1,78 @@
+# Deploying ASO v2 in multi-tenant mode
+
+The current release version of the ASO deployment YAML installs the operator in single-tenant mode:
+it's deployed in the `azureserviceoperator-system` namespace with one set of Azure credentials, and manages resources in any namespace in the cluster.
+That single operator deployment handles webhooks fired when Azure resources are changed.
+
+ASO can also be deployed in a multi-tenant configuration that enables using separate credentials to manage resources in different Kubernetes namespaces.
+
+Running the operator in multi-tenant mode requires one deployment to handle webhooks (since webhook configurations are cluster-level resources) and then a deployment for each tenant, each with its own credentials and set of namespaces that it watches for Azure resources.
+To deploy the operator in multi-tenant mode the release YAML has been split into two parts:
```

Comment:
Minor suggestions for clarity.

```suggestion
Running the operator in multi-tenant mode uses one deployment to handle webhooks (required because webhook configurations are cluster-level resources), and then a separate deployment for each tenant, each with its own credentials and a set of namespaces that it watches for Azure resources.

To deploy the operator in multi-tenant mode the release YAML has been split into two parts:
```